addons/binding/org.openhab.binding.dscalarm/README.md

```diff
@@ -168,7 +172,7 @@ DSC Alarm things support a variety of channels as seen below in the following ta
     <tr><td>keypad_lcd_cursor</td><td>String</td><td>LCD Cursor Position for The IT-100</td></tr>    
 </table>
 
-## Full Example
+**Example**
```

Comment:
Why is this not a section anymore?

Reply:
Agree - will restore as section in my next commit.